### ISSUE TITLE
Cannot login to Sokowatch website

### ISSUE THREAD

**Issue by cypru55** [Reliefwatch/wrigley_guide#16]

**Comment by dcyu** [comment]
@cypru55 Thanks for posting up the issue about problems logging in. We just noticed that as well and while looking into the problem have noticed that some of the table rows have been dropped. We should have backups to be able to restore most of them, but I want to double check to make sure we're not missing anything.

Do you have the sales data CSV files by month? If you wouldn't mind sending those along to me so I can double check that would be great.

**Comment by cypru55** [comment]
@dcyu I only sure that the data I have is same with your website only until nov 2015.
Dec 2015 and Jan 2016 might not be the latest one because when I downloaded for the incentive payment, and when I downloaded again at different time there are some differences. When is your latest backup data up to?

I can't upload csv file, I put as xlsx and all data is combined until 25th Jan 2016.
[sales_data.xlsx](https://github.com/Reliefwatch/wrigley_guide/files/140701/sales_data.xlsx)
